user_manual/talk/advanced_features.rst

```diff
@@ -251,4 +252,13 @@ You can generate it by pressing the button that is visible in chat above the fir
 
 .. image:: images/chat-summary-text.png
    :width: 500px
+
+Meeting scheduling from Talk
```

Comment:
I just added a section for meetings in the last pr. Shall we join these sections?